tdrs-frontend/package.json

```diff
@@ -34,7 +34,8 @@
   "resolutions": {
     "lodash": "^4.17.20",
     "bl": "^4.0.3",
-    "node-forge": "0.10.0"
+    "node-forge": "0.10.0",
+    "object-path": "0.11.5"
```

Comment:
`object-path` @ 0.11.4 was throwing a Snyk vulnerability. This fixes the vulnerability.